packages/defaults.yaml

```diff
@@ -22,7 +22,11 @@ packages:
       states: []
       pkgs: []
   snaps:
-    package: snapd
+    {% if  grains['os'] == 'Ubuntu' %}
+    packages: ['snapd', 'fuse',]
```

Comment:
This values should be set in the [`osmap.yaml`](https://github.com/saltstack-formulas/packages-formula/blob/master/packages/osmap.yaml) file, like:

```yaml
Ubuntu:
  snaps:
    packages: ['snapd', 'fuse']
```
and keep `defaults.yaml` as it was, with no conditional, to keep it clean.

Reply:
Yes, the conditional should go.  AFAIK these package names are valid for any Linux distribution, and not just ubuntu so `defaults.yaml` is correct place.  Note Centos has no snapd package available.

Reply:
Maybe package names are valid for any Linux distribution, but they aren't needed for all of them. For example, `Debian` build just fine without `fuse` package.